ietf/doc/views_help.py

```diff
@@ -9,6 +9,18 @@
 from ietf.name.models import DocRelationshipName,  DocTagName
 from ietf.doc.utils import get_tags_for_stream_id
 
+def state_index(request):
+    types = StateType.objects.all()
+    names = [ type.slug for type in types ]
+    for type in types:
+        if "-" in type.slug and type.slug.split('-',1)[0] in names:
+            type.stategroups = None
+        else:
+            groups = StateType.objects.filter(slug__startswith=type.slug)
+            type.stategroups =  [ g.slug[len(type.slug)+1:] for g in groups if not g == type ] or ""
+                
+    return render(request, 'doc/state_index.html', {"types": types})
```

Comment:
the concept of `used` on State probably needs to be rendered here somewhere - maybe that can be done entirely in the templates.

Reply:
nevermind - the bit is badly used in the State table.